src/ClassApplication/Controllers/ClassController.cs

```diff
@@ -163,6 +163,37 @@ public async Task<ActionResult<string>> UpdateClass(string classId, string class
                 return BadRequest();
         }
 
+        public async Task<ActionResult<string>> RemoveStudent(string classId, string actorId, string removeId)
+        {
+
+            Classroom classroom;
+            classroom = await classesContainer.ReadItemAsync<Classroom>(classId, partitionKey);
+
+            // if either the actor or the user to be removed are not part of this classroom,
+            //      this is a bad request
+            if (!classroom.ContainsStudent(actorId) || !classroom.ContainsStudent(removeId))
+                return BadRequest();
+
+            // if the user being removed is the Owner
```

Comment:
Good job with this function! But here's another case to consider: 
What if a student, who's not an owner, wants to remove themselves from a class? Should this be allowed?